tests/unit/components/footer/ApplicationFooter.test.js

```diff
@@ -0,0 +1,28 @@
+import { installQuasarPlugin } from '@quasar/quasar-app-extension-testing-unit-vitest';
+import { mount } from '@vue/test-utils';
+import {
+  describe,
+  expect,
+  it,
+  beforeEach,
+  vi,
+} from 'vitest';
+import ApplicationFooter from 'src/components/footer/ApplicationFooter.vue';
+import { createPinia, setActivePinia } from 'pinia';
+
+installQuasarPlugin();
+
+vi.stubGlobal('$sanitize', true);
+
+describe('Test component: ApplicationFooter', () => {
+  let wrapper;
+
+  beforeEach(() => {
+    setActivePinia(createPinia());
+    wrapper = mount(ApplicationFooter);
+  });
+
+  it('should mount the component', () => {
```

Comment:
should check `version` ref initialization

Reply:
no, because we never check ref. Only check computed.